packages/fxa-auth-server/lib/routes/subscriptions/stripe.ts

```diff
@@ -420,6 +421,22 @@ export class StripeHandler {
     return stripeInvoiceToInvoicePreviewDTO(previewInvoice);
   }
 
+  async retrieveCouponDetails(request: AuthRequest): Promise<any> {
```

Comment:
No need to use `any`.  We know the type here and it is the type of `couponDetails`.

Reply:
updated